src/rng/CMakeLists.txt

```diff
@@ -3,93 +3,85 @@
 # author Kelly Thompson <kgt@lanl.gov>
 # date   2012 Aug 1
 # brief  Generate build project files for rng.
-# note   Copyright (C) 2010-2019, Triad National Security, LLC.
+# note   Copyright (C) 2010-2020, Triad National Security, LLC.
 #        All rights reserved.
 #------------------------------------------------------------------------------#
-cmake_minimum_required(VERSION 3.9.0)
+cmake_minimum_required(VERSION 3.17.0)
 project( rng CXX )
 
+# Local variables -------------------------------------------------------------#
+string(CONCAT header_prefix_source 
+  "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>"
+  "$<INSTALL_INTERFACE:${DBSCFG_IMPORT_PREFIX}/rng>")
+string(CONCAT header_prefix_binary "$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}>"
+  "$<INSTALL_INTERFACE:${DBSCFG_IMPORT_PREFIX}>")
+
 # ---------------------------------------------------------------------------- #
 # Generate config.h (only occurs when cmake is run)
 # ---------------------------------------------------------------------------- #
-# default to using C++11 features of Random123.
-get_target_property( cxxstd Lib_dsxx CXX_STANDARD )
-if( cxxstd AND NOT ${cxxstd} STREQUAL 98 )
-  set( R123_USE_CXX11 1 )
-endif()
+set( R123_USE_CXX11 1 )
```

Comment:
So this change means we always use the c++11 features of r123 now? Presumably because we won't be using any compilers that aren't at least c++11 compliant :)

Reply:
We can't compile without at least C++14 support, so this check was meaningless.